src/main/java/uk/gov/pay/directdebit/events/exception/InvalidGovUkPayEventInsertionException.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.pay.directdebit.events.exception;
+
+import uk.gov.pay.directdebit.events.model.GovUkPayEvent;
+
+import static java.lang.String.format;
+
+public class InvalidGovUkPayEventInsertionException extends RuntimeException {
```

Comment:
I don't feel too strongly about this either way, the only reason I went for the static method pattern was to make it clearer that when just the one event was provided the exception would state that this was the initial event. If you think it makes it less clear I'm happy to change.

Reply:
Ah I see, perhaps if its important to distinguish between those two scenarios then it might be better to have them as two separate exceptions if they're not actually the same thing, otherwise I'd favour the simpler constructor approach.